src/pages/AddPatient/AddPatient.js

```diff
@@ -61,25 +122,55 @@ const AddPatient = () => {
 
   return (
     <DashboardLayout>
-      <div className="header mb-1 d-flex justify-content-between px-3 py-2">
-        <h3 className="page-title">Add New Patient</h3>
-      </div>
+      <Headings>
+        <InfoWrapper>
+          <ViewName>Add New Patient</ViewName>
+          <DateAndTimeWrap>
+            <TimeImage src={time} />
+            <DateAndTime>{getDate()}</DateAndTime>
+          </DateAndTimeWrap>
+        </InfoWrapper>
+      </Headings>
       <Container>
         <Form onSubmit={handleSubmit(handleSave)}>
           <Row>
             <Col md={{ size: 6 }}>
-              <InputField
-                title="Cell Phone"
-                name="phone"
-                required
-                error={getErrorMessage(errors, 'phone')}
-                innerRef={register}
-                defaultValue="+91-"
-              />
+              <Label className="required w-100">Cellphone Number</Label>
+              <div className="d-flex">
+                <Col md={{ size: 3 }} className="px-0">
+                  <Dropdown
+                    isOpen={isPhoneCodesDropdownOpen}
+                    toggle={togglePhoneCodesDropdown}
+                    direction="down"
+                    className="classic-dropdown">
+                    <DropdownToggle caret className="w-100">
+                      {phoneCode || 'Code'}
+                    </DropdownToggle>
+                    <DropdownMenu>
+                      {countries.map((country) => (
+                        <DropdownItem
+                          key={getRandomKey()}
```

Comment:
it would be better to just use `country.phone_code` as `key` instead of calling this extra function

Reply:
fixed